libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -65,33 +65,33 @@ export class DaffCartFacade implements DaffStoreFacade<Action> {
   availablePaymentMethods$: Observable<DaffCart['available_payment_methods']>;
   isCartEmpty$: Observable<boolean>;
 
-  constructor(private store: Store<DaffCartReducersState>) {
-    this.loading$ = this.store.pipe(select(selectCartLoading));
-    this.cart$ = this.store.pipe(select(selectCartValue));
+  constructor(private store: Store<DaffCartReducersState<T>>) {
+    this.loading$ = this.store.pipe(select(selectCartLoading()));
```

Comment:
Should you be passing `T` into the selectors?

Reply:
I don't think so. Any selector daffodil defines will only ever access the base `DaffCart` model. It shouldn't ever need to know specifically what the model is. Just that it is `T extends DaffCart`.

Reply:
I changed this to include `T` for selectors that need it